### DISCUSSION TITLE
Latexmk: Maximum runs of xelatex reached without getting stable files.

### DISCUSSION THREAD

**Discussion by chuyan99** [sjtug/SJTUThesis#1075] [提问]
用Macos+Texstudio编译时，到后面参考文献超过220多个后，新加入的文献一直编码不出来，报这个错，请问是因为文献太多的原因吗？如何解决

**Comment by chuyan99** [comment]
排查过bib文件和cite部分是没有错的。不加入新的章节部分，只编码前面也可以正常运行；把中间几章注释后，新加入的章节文献也可以正常编码。但全加一起就会崩。

**Reply by LogCreative** [reply]
好像没有看到报错的图片（或者日志信息）。

**Comment by hushidong** [comment]
biber 不会有处理文献数量的限制。
大概率还是bib文件存在问题。
你用 biber  main 命令看看，是什么输出？贴上来。

**Comment by chuyan99** [comment]
biber main: INFO - This is Biber 2.19
INFO - Logfile is 'main.blg'
INFO - Reading 'main.bcf'
INFO - Found 229 citekeys in bib section 0
INFO - Processing section 0
INFO - Looking for bibtex file 'refs.bib' for section 0
INFO - LaTeX decoding ...
INFO - Found BibTeX data source 'refs.bib'
INFO - Overriding locale 'en-US' defaults 'normalization = NFD' with 'normalization = prenormalized'
INFO - Overriding locale 'en-US' defaults 'variable = shifted' with 'variable = non-ignorable'
INFO - Sorting list 'none/global//global/global' of type 'entry' with template 'none' and locale 'en-US'
INFO - No sort tailoring available for locale 'en-US'
INFO - Writing 'main.bbl' with encoding 'UTF-8'
INFO - Output to main.bbl
WARN - BibTeX subsystem: /var/folders/mn/jf6355bs5k9_j2cprkw056x00000gn/T/biber_tmp_CJwF/8fe4c9854184545302f21ebf0d47516c_67592.utf8, line 612, warning: 1 characters of junk seen at toplevel
INFO - WARNINGS: 1

**Comment by chuyan99** [comment]
运行main.tex 报错： Latexmk: ====Undefined refs and citations with line #s in .tex file: Citation 'sirinukunwattana2016locality' on page 1 undefined on input line 9 Citation 'litjens2017survey' on page 1 undefined on input line 9 Citation 'pan2009survey' on page 1 undefined on input line 12 Citation 'bengio2012deep' on page 1 undefined on input line 12 Citation 'choudhary2021chest' on page 1 undefined on input line 12 Citation 'chen2018semantic' on page 1 undefined on input line 12 Citation 'bhardwaj2021transfer' on page 1 undefined on input line 12 And 611 more --- see log file 'main.log'
Latexmk: Sometimes, the -f option can be used to get latexmk to try to force complete processing. But normally, you will need to correct the file(s) that caused the error, and then rerun latexmk. In some cases, it is best to clean out generated files before rerunning latexmk after you've corrected the files.
出现错误

**Comment by hushidong** [comment]
从biber结果看，bib文件第612行有问题，你查一下。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
